docs/Getting Started.md

```diff
@@ -0,0 +1,171 @@
+# Getting Started
+Welcome to Inductiva, where large-scale simulations meet simplicity, through a 
+seamless, user-friendly interface that brings cutting-edge computational resources 
+to your fingertips—effortlessly.
+
+This section is designed to provide you with all the necessary information you 
+need to get started with Inductiva API. Here, we've condensed everything so you 
+can install Inductiva, grab usage tips, and find some of its essential features, 
+with ease.
+
+It's all you need to get familiar and make the most out of Inductiva from the 
```

Comment:
I would also remove this sentence for now. Let's go straight to the point.

Reply:
for a nice comparison, you can check the style/tone of this section vs the home page iteration PR: https://github.com/inductiva/inductiva/pull/983
This one has a completely different style, so we can pick and choose what we like!

Reply:
This is great, because of the Genesis program (I will tell you what it is) we will need this style. Don't loose track of this style because we will need it elsewhere.